lib/services/geolocation_service.dart

```diff
@@ -0,0 +1,49 @@
+import "package:cloud_firestore/cloud_firestore.dart";
+import "package:geolocator/geolocator.dart";
+
+class GeoLocationService {
+  final GeolocatorPlatform _geoLocator;
+
+  // Settings to get the most accurate location
```

Comment:
Yes you are right, it makes more sense to use `LocationAccuracy.best` for our setting.
The commit 2af73d94327d29b8c97af7cafeb13524db45e11a fix that.

Reply:
Thanks!
What about the comment "Settings to get the most accurate location", though? I have to admit I still consider it is not correct. In my opinion, explaining your choice of `best` over `high` and `bestForNavigation` in it would be better.